font-codegen/src/table.rs

```diff
@@ -738,3 +786,22 @@ impl TableReadArgs {
         })
     }
 }
+
+// An overwrought and likely incorrect way of converting 'Format1' to 'format_1' -_-
```

Comment:
Very unit testable :) Would like to see more unit tests in our CLs in general.

This looks like a manual implementation of a regex, e.g. replace letter-number with letter-dash-number, which could make sense for perf in some situations but ... probably not this one?